### ISSUE TITLE
[JS][Visualizer] Visualizer doesn't distinguish Action sentiments

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#2884]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* JavaScript

# Issue
 The [visualizer](https://adaptivecards-ci.azurewebsites.net/visualizer/index.html) doesn't render actions with Positive and Destructive sentiments differently than default actions. The designer does have different styling for these sentiments.

Designer:
![image](https://user-images.githubusercontent.com/9091833/57556048-c60be600-732a-11e9-8641-6a30b9b081f1.png)

Visualizer:
![image](https://user-images.githubusercontent.com/9091833/57555990-9957ce80-732a-11e9-8369-d72272318d05.png)

Json:
```
{
	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
	"type": "AdaptiveCard",
	"version": "1.2",
	"actions": [
		{
			"type": "Action.OpenUrl",
			"sentiment": "default",
			"url": "http://adaptivecards.io",
			"title": "Default"
		},
		{
			"type": "Action.OpenUrl",
			"sentiment": "positive",
			"url": "http://adaptivecards.io",
			"title": "Positive"
		},
		{
			"type": "Action.OpenUrl",
			"sentiment": "destructive",
			"url": "http://adaptivecards.io",
			"title": "Destructive"
		},
		{
			"type": "Action.OpenUrl",
			"sentiment": "other",
			"url": "http://adaptivecards.io",
			"title": "Other"
		}
	]
}
```

**Comment by shalinijoshi19** [comment]
@dclaux is this something we need to fix for 1.2?

**Comment by dclaux** [comment]
Won't fix. The visualizer will not be updated any more going forward, and will eventually be retired as its functionality is moved to the designer.
